allensdk/brain_observatory/ecephys/_channels.py

```diff
@@ -17,9 +18,17 @@ def __init__(self, channels: List[Channel]):
         super().__init__(name='channels', value=channels)
 
     @classmethod
-    def from_json(cls, channels: dict) -> "Channels":
-        channels = [Channel(**{f'{"impedance" if k == "impedence" else k}': v
-                               for k, v in channel.items()})
+    def from_json(
+            cls,
+            channels: dict
+    ) -> "Channels":
+        for channel in channels:
+            if 'impedence' in channel:
+                # Correct misspelling
+                channel['impedance'] = channel.pop('impedence')
+
+        channels = [Channel(**{k: v for k, v in channel.items()
```

Comment:
Would it be possible to explicitly pass in the args/kwargs for `Channel`. I'm not a fan of how obfuscated this is.

If the number of optional parameters in `channel.items()` make this fix too much work, I'm willing to give it a pass.

Reply:
Sure, I've explicitly passed in the arguments